content/en/docs/pluggable-components/service-mesh.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Enable Service Mesh"
+keywords: "Kubernetes, istio, KubeSphere, service-mesh, microservices"
+description: "How to Enable KubeSphere Service Mesh"
+
+linkTitle: "Enable Service Mesh"
```

Comment:
title: KubeSphere Service Mesh System
linkTitle: KubeSphere Service Mesh System

Reply:
I changed the title as I think it may be better to say Enable Service Mesh directly as the section talks about how to enable it instead of the feature itself.

Reply:
If you think `Enable` is necessary within each title, you can keep it like `Enable KubeSphere Service Mesh System`